src/Symfony/Component/Notifier/Bridge/Iterable/README.md

```diff
@@ -0,0 +1,21 @@
+# Iterable Notifier
+
+Provides [Iterable](https://iterable.com/) integration for Symfony Notifier.
```

Comment:
ohh :-( when reading the title of this PR, I thought it was about a special provider that broadcast (or fallback) over a list of providers ..